libraries/src/test/java/com/baeldung/mapdb/CollectionsUnitTest.java

```diff
@@ -0,0 +1,33 @@
+package com.baeldung.mapdb;
+
+import org.junit.Test;
+import org.mapdb.DB;
+import org.mapdb.DBMaker;
+import org.mapdb.Serializer;
+
+import java.util.NavigableSet;
+
+import static junit.framework.Assert.assertEquals;
+
+public class CollectionsUnitTest {
+
+  @Test
+  public void givenSetCreatedInDB_whenMultipleElementsAdded_checkOnlyOneExists() {
+
+    DB db = DBMaker.memoryDB().make();
+
+    NavigableSet<String> set = db.
+            treeSet("mySet")
+            .serializer(Serializer.STRING)
+            .createOrOpen();
+
+    final String MY_STRING = "Baeldung!";
```

Comment:
Let's take the `final` keyword out altogether. It's really not a good keyword unless you're trying to do something special.

In this case you have a local variable, which you're not changing and you're trying to write it as though it's a class level constant.

`private static final String MY_STRING` would be a fine thing to have for a global constant, but that's not what's happening here.

Reply:
Let's call it `myString`

Reply:
Changed MY_STRING to myString and dropped the final